scss/blocks/figma-component-props-table.scss

```diff
@@ -67,7 +68,7 @@ table.props-table .props-table-value {
     font-family: "IBM Plex Mono", monospace;
     font-size: 14px;
     line-height: 20px;
```

Comment:
In `typography.scss`, we have `mark` definition using this:

```
font-family: 'Fira Mono', 'Fira Code', Menlo, Consolas, 'DejaVu Sans Mono', monospace;
font-weight: 600;
font-size: 0.825em;
```

It would be great not to import another monospace font for it + probably create new variables e.g. `fontFamilyMono`  and share the family definition better for those cases

Reply:
But in the Figma we have IBM Plex Mono, is it ok to switch it to Fira Mono then?

Reply:
Yes please